src/Helpers.js

```diff
@@ -165,6 +165,12 @@ export const TRIGGER_PREFIX_BELOW = "<";
 
 export const MIN_PROFIT_BIPS = 0;
 
+// USD tokens per interval given by kurtis
+export const MM_TOKENS_PER_INTERVAL = ethers.utils.parseUnits("0.1859475633", USD_DECIMALS);
+export const FEE_MULTIPLIER_BASIS_POINTS = 4;
+export const MM_FEE_MULTIPLIER = bigNumberify(6);
+export const MM_SWAPS_FEE_MULTIPLIER = bigNumberify(12);
+
```

Comment:
I thought we removed this stuff.  Is this part of the PR or did it sneak back in?

Reply:
Snuck back in - need me to remove?